Assets/MixedRealityToolkit/Utilities/Editor/FileUtilities.cs

```diff
@@ -13,14 +14,35 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
     public static class FileUtilities
     {
         /// <summary>
-        /// Locates the files that match the specified name within the Assets folder (application data path) structure.
+        /// Locates the files that match the specified name within the Assets folder structure.
         /// </summary>
         /// <param name="fileName">The name of the file to locate (ex: "TestFile.asmdef")</param>
+        /// <param name="searchFullFileSystem">If true, searches the full application data path. If false, only searches Unity's known asset database.</param>
         /// <returns>Array of FileInfo objects representing the located file(s).</returns>
-        public static FileInfo[] FindFilesInAssets(string fileName)
+        public static FileInfo[] FindFilesInAssets(string fileName, bool searchFullFileSystem = false)
```

Comment:
shouldn't the default be true as this was the original behavior?

Reply:
Since this method hasn't shipped in a release yet, I decided to change the default while leaving the existing behavior around, since I think it can be useful in some cases.
I can flip it around though! I do think the "asset database only" cases feels like the more common one to me (and thus made it the default), but I'm good either way.
I chatted with @davidkline-ms about this a bit, so tagging him here in case he has other thoughts.

Reply:
@thalbern, i suggested @keveleigh default to false as this is a better default behavior (it allows for honoring Unity's hiding of the .<folder> pattern.

Reply:
Yeah as long as we haven't already put this method in an existing release, we're fine to change as we want. 

I would even be open to not having the alternate behavior and introducing when needed (because we don't currently have a test for the alternate behavior, it could grow stale and not work)